UnityProjects/MRTKDevTemplate/Packages/manifest.json

```diff
@@ -1,7 +1,5 @@
 {
   "dependencies": {
-    "com.atteneder.gltfast": "https://github.com/atteneder/glTFast.git#v4.8.3",
-    "com.atteneder.ktx": "https://github.com/atteneder/KtxUnity.git#v2.1.2",
```

Comment:
Ah, looks like there's a git LFS bandwidth limit reached 😬 interesting...

![image](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/3580640/d311b9c3-fa89-4653-a044-44a6445d3422)

I'm okay removing this from our sample project for now, but we should add a note somewhere that it's required for controller models on Quest devices.

Reply:
Ah, one more thought on this: this is probably a good time to remove all the `KTX_PRESENT` references throughout (including the declarations on the asmdefs), since we don't _actually_ have a code dependency on the package, just a functional one on Quest devices

> but we should add a note somewhere that it's required for controller models on Quest devices.

Thinking about this more, the MRTK suggestion window thing should be sufficient for this, iirc.

Reply:
@ms-RistoRK , I think we should leave the GLTF package in the MRTK3 sample.

Reply:
I believe to have attended the requested changes, please re-review.